src/style.css

```diff
@@ -0,0 +1,55 @@
+* {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 140px;
+  height: 160px;
```

Comment:
No need to set container size here, it should depend on size and distance between elements.
distance should be taken from figma mockup.